alacritty_terminal/src/term/mod.rs

```diff
@@ -1156,23 +1152,25 @@ impl<T> Term<T> {
         debug!("New num_cols is {} and num_lines is {}", num_cols, num_lines);
 
         let is_alt = self.mode.contains(TermMode::ALT_SCREEN);
-        let alt_cursor_point =
-            if is_alt { &mut self.cursor_save.point } else { &mut self.cursor_save_alt.point };
 
         // Resize grids to new size
         self.grid.resize(!is_alt, num_lines, num_cols, &mut self.cursor.point, &Cell::default());
-        self.alt_grid.resize(is_alt, num_lines, num_cols, alt_cursor_point, &Cell::default());
+        self.inactive_grid.resize(
+            is_alt,
+            num_lines,
+            num_cols,
+            &mut self.inactive_cursor.point,
+            &Cell::default(),
+        );
```

Comment:
I feel like it might be worth extracting `&mut self.inactive_cursor.point` into a variable here to prevent this linewrap.